### ISSUE TITLE
Home Page - Missing ID on INPUT Field - Improperly Coded FORM LABEL and INPUT Field

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#4]
While testing using DHS Trusted Tester process, I found there was no ID on the INPUT field for the drop down list which makes it inaccessible to those using a screen reader or less so for those using the mouse to be able to click on the label instead of INPUT filed. There is a FOR on the LABEL (I am a:), but ID is missing on the INPUT field, which is a drop down list.

Section 508 - 22(n)
WCAG 2.0 - 1.2.2 Web: Forms

**Comment by jfc3-dol** [comment]
Here are two screenshots showing the LABEL with a FOR and NO ID:

![worker dot gov - label that has for but no id on input](https://cloud.githubusercontent.com/assets/10423487/19775068/2eb92a2c-9c3d-11e6-8520-8e896c1c8e3b.png)

And here is the INPUT field with no ID:

![worker dot gov - input field with no id to connect to label](https://cloud.githubusercontent.com/assets/10423487/19775081/3c0a8810-9c3d-11e6-8089-db9d3a2e4d2d.png)
